index.html

```diff
@@ -208,7 +208,7 @@ <h3>Skills</h3>
     <footer>
         <div class="bottom">
             <h2 class="white">Contact me </h2>
-            <p class="call">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <p class="call">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
             <form action="https://formspree.io/f/xkneznwb"
             method="POST">
     
```

Comment:
### This is a CSS styling issue 💅🏼 

- [OPTIONAL] please consider fixing the position of the form submit button as shown in the Figma design

#### Your Design
![image](https://user-images.githubusercontent.com/68030297/221371652-725291e1-be97-45a3-aa67-d80d0adc0531.png)

#### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221371712-adeaca44-9e93-4e52-a795-9689fbcfb6c6.png)